src/components/Exchange/ConfirmationBox.css

```diff
@@ -0,0 +1,28 @@
+.trigger-order-list {
+  list-style: none;
+  padding: 0;
+  margin: 0.5rem 0 1rem;
+}
+.trigger-order-list li{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: var(--dark-blue-hover);
+  border-radius: var(--border-radius-sm);
+  margin-bottom: .25rem;
+  padding-bottom: 0;
+  padding: .5rem;
+}
+.trigger-order-list li p{
```

Comment:
we don't have styleguides for css but let's keep space before `{`? :)

Reply:
Sure!